qiskit/providers/models/pulsedefaults.py

```diff
@@ -94,44 +98,335 @@ class Command(BaseModel):
     full description of the model, please check ``CommandSchema``.
 
     Attributes:
-        name (str): Pulse command name.
+        name: Pulse command name.
     """
-    def __init__(self, name, **kwargs):
+    def __init__(self, name: str, **kwargs):
         self.name = name
 
         super().__init__(**kwargs)
 
 
 @bind_schema(PulseDefaultsSchema)
 class PulseDefaults(BaseModel):
-    """Model for PulseDefaults.
-
-    Please note that this class only describes the required fields. For the
-    full description of the model, please check ``PulseDefaultsSchema``.
-
-    Attributes:
-        qubit_freq_est (list[number]): Estimated qubit frequencies in GHz.
-        meas_freq_est (list[number]): Estimated measurement cavity frequencies
-            in GHz.
-        buffer (int): Default buffer time (in units of dt) between pulses.
-        pulse_library (list[PulseLibraryItem]): Backend pulse library.
-        cmd_def (list[Command]): Backend command definition.
+    """Description of default settings for Pulse systems. These are operations or settings that
+    may be good starting points for the Pulse user. The user may modify these defaults through
+    the provided methods to build a reference to custom operations, which may in turn be used
+    for building Schedules or converting circuits to Schedules.
     """
 
-    def __init__(self, qubit_freq_est, meas_freq_est, buffer,
-                 pulse_library, cmd_def, **kwargs):
-        self.qubit_freq_est = qubit_freq_est
-        self.meas_freq_est = meas_freq_est
+    def __init__(self,
+                 qubit_freq_est: List[float],
+                 meas_freq_est: List[float],
+                 buffer: int,
+                 pulse_library: List[PulseLibraryItem],
+                 cmd_def: List[Command],
+                 **kwargs: Dict[str, Any]):
+        """
+        Validate and reformat transport layer inputs to initialize.
+
```

Comment:
Could we add a todo that these should be initializable without transport in the future?

Reply:
that's outside the scope of a TODO. it would overhaul the structure and usage of `qiskit/providers`, not only this one class, and I think (hope) it can be considered IP with what mattT has been looking into